.husky/pre-push

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
-# this only for targeting canary at the moment 
-# fetching latest changes to ensure the origin/canary is up to date
+# this only for targeting arc-themes-release-version-1.19 at the moment 
```

Comment:
These changes are in another PR too - should we create a separate PR to address the issue/needs for this?